### DISCUSSION TITLE
[llama-server] Please show me how to increase stream updates frequency

### DISCUSSION THREAD

**Discussion by ghost** [ggml-org/llama.cpp#9207] [Q&A] [answered]

**Comment by ExtReMLapin** [comment]
On my end, manually starting llama.cpp server with command line, it's updated on each tokens, stream being sent in body of the POST request.

**Comment by ghost** [comment, marked as answer]
I did a bit more research, and was able to locate the issue. It was nginx's fault. Adding the following to my configuration solved my issue:
```
proxy_buffering off;
proxy_cache off;
```
